### PR TITLE
Add websocket source

### PR DESCRIPTION
**Pull request by emctoo**

### CODE REVIEW

**Comment by xoolive** [comment]
Related to #47 

Just thinking from the top of my head:
- maybe an Enum is better for the scheme, maybe add TCP_UDP (if empty, then try TCP before UDP)
- some parsing error would be nice (e.g. pattern should be empty if TCP or UDP)

Will check asap

**Comment by xoolive** [comment]
The code still does not fully compile (I didn't get what you want to do with `name`) but I took out most of the errors

**Comment by xoolive** [comment]
Now we have a working implementation, happy to press merge anytime.
Can you confirm it works on your side?

File: crates/jet1090/src/cli.rs [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+use radarcape::Address;
 use rs1090::decode::{cpr::Position, TimedMessage};
 use rs1090::prelude::*;
 use serde::{Deserialize, Serialize};
 use tokio::sync::mpsc::Sender;
+use tracing::error;
+use url::Url;
+
+#[derive(Debug, Clone, Default, PartialEq, Serialize, Deserialize)]
+pub enum Protocol {
```
Comment (xoolive) [inline crates/jet1090/src/cli.rs:12]: Since `host` and `port` don't make sense for `Rtlsdr`, maybe it should be `Tcp(String)` and `Udp(String)` as well...

**Comment by emctoo** [comment]
@xoolive Yeah, it's working. Well done!

**Pull request merged**